<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourSetting</class>
 <widget class="QDialog" name="ContourSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>161</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>161</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting Colour Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number of Contours     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nocLine">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Contour Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="positionCombo">
           <item>
            <property name="text">
             <string>Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bottom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Font-size                       </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fontSizeLine">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Numeric type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="numericCombo">
           <item>
            <property name="text">
             <string>Float</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Interger</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scientific</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="nodeCheckBox">
           <property name="text">
            <string>Node View</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="meshCheckBox">
           <property name="text">
            <string>Undeformed mesh</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="deformMeshCheckBox">
           <property name="text">
            <string>Deformed mesh</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="resultCheckBox">
           <property name="text">
            <string>Results View</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="axeCheckBox">
           <property name="text">
            <string>Axe View</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="dateCheckBox">
           <property name="text">
            <string>Date</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="titleCheckBox">
           <property name="text">
            <string>Title</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="titleLine">
           <property name="text">
            <string>Unit is m, s, kPa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maxCheckBox">
           <property name="text">
            <string>Max, Min Values</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Apply/OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
